src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -613,12 +538,18 @@ private InstructorFeedbackResultsGroupByParticipantPanel buildGroupByParticipant
                     && instructor.isAllowedForPrivilege(response.recipientSection,
                                                         response.feedbackSessionName,
                                                         Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS);
-            
+
+            Pattern pattern = Pattern.compile("^section-(\\d+)");
```

Comment:
Same point as the previous round of review, `Pattern` can be made into a static field?